spark/src/main/scala/ai/chronon/spark/Driver.scala

```diff
@@ -275,6 +275,68 @@ object Driver {
     }
   }
 
+  object JoinBackfillLeft {
+    @transient lazy val logger = LoggerFactory.getLogger(getClass)
+    class Args
+        extends Subcommand("join-left")
+        with OfflineSubcommand
+        with LocalExportTableAbility
+        with ResultValidationAbility {
+      val runFirstHole: ScallopOption[Boolean] =
+        opt[Boolean](required = false,
+                     default = Some(false),
+                     descr = "Skip the first unfilled partition range if some future partitions have been populated.")
+      val startPartitionOverride: ScallopOption[String] =
+        opt[String](required = false,
+                    descr =
+                      "Start date to compute join backfill, this start date will override start partition in conf.")
```

Comment:
these two params have been refactored into the offlinecommand. They are not needed any more here. 
```suggestion
```